src/schemas/application.ts

```diff
@@ -0,0 +1,119 @@
+import z from "zod";
+
+const schema = z.object({
+  firstName: z.string().min(1).max(255),
+  lastName: z.string().min(1).max(255),
+  birthday: z.string().refine(
+    (value) => {
+      // parse date
+      const date = new Date(value);
```

Comment:
does this take into account timezones? i assume serverside date is stored in UTC?